.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - SFDX_AUTOUPDATE_DISABLE=true
     - SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
     - SFDX_DOMAIN_RETRY=300
+    - CODE_TESTS_WORKSPACE=$TRAVIS_BUILD_DIR/packages/system-tests/sfdx-simple
```

Comment:
Ok I can change it so that it uses it's own workspace, it defaults to `process.cwd()/out/test` which means that the workspace won't be top level but that is fine for the cases I'm testing.